addons/CLib/Core/Misc/fn_clientInitModCheck.sqf

```diff
@@ -0,0 +1,78 @@
+#include "macros.hpp"
+/*
+    Community Lib - CLib
+
+    Author: joko // Jonas
+
+    Description:
+    Server for Core Module
+
+    Parameter(s):
+    None
+
+    Returns:
+    None
+*/
+
+
+DFUNC(clearAddons) = {
+    params ["_addons", "_ignore"];
+
+    _addons params ["_name", "_version"];
+    {
+        if (_x in _ignore) then {
+            _name set [_forEachIndex, objNull];
+            _version set [_forEachIndex, objNull];
+        };
+    } forEach (_addons select 0);
+
+    _name = _name - [objNull];
+    _version = _version - [objNull];
+
+    [_name, _version]
+};
+
+
+[{
+    GVAR(serverAddons) params ["_allServerAdons", "_ignoredAddons"];
+
+    _ignoredAddons append getArray (missionConfigFile >> "CLib_IgnoredAddons");
+
+    _allServerAdons = _allServerAdons call FUNC(clearAddons);
+
+    _allServerAdons params ["_namesServer", "_versionsServer"];
+
+    private _loadedAddons = call FUNC(getAllAddons);
+
+    _loadedAddons = _loadedAddons call FUNC(clearAddons);
```

Comment:
missing parameter: ignored addons

edit:
this call can be omitted if ignored addons are processed serverside